### PR TITLE
RDSv3: Migrate to gopher sdk

### PR DESCRIPTION
**Pull request by lego963**
## Summary of the Pull Request

Refactoring RDSv3 resource and migrate it to SDK from directly in resource realization.

## PR Checklist

* [x] Refers to: #594
* [x] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
=== RUN   TestAccRdsInstanceV3_basic
--- PASS: TestAccRdsInstanceV3_basic (1141.79s)
=== RUN   TestAccRdsInstanceV3_ip
--- PASS: TestAccRdsInstanceV3_ip (690.61s)
=== RUN   TestAccRdsInstanceV3_ha
--- PASS: TestAccRdsInstanceV3_ha (526.50s)
PASS

Process finished with exit code 0
```

### CODE REVIEW

**Comment by lego963** [comment]
Delete method requires changes in SDK

**Comment by outcatcher** [comment]
recheck

File: opentelekomcloud/resource_opentelekomcloud_rds_instance_v3.go [hunk 1]
```diff
@@ -161,31 +154,50 @@ func resourceRdsInstanceV3() *schema.Resource {
...
 			},
-
 			"ha_replication_mode": {
 				Type:     schema.TypeString,
 				Computed: true,
 				Optional: true,
 				ForceNew: true,
 			},
-
+			"switch_strategy": {
```
Comment (outcatcher) [inline opentelekomcloud/resource_opentelekomcloud_rds_instance_v3.go]: Maybe move this to another PR?

File: opentelekomcloud/resource_opentelekomcloud_rds_instance_v3.go [hunk 2]
```diff
@@ -161,31 +154,50 @@ func resourceRdsInstanceV3() *schema.Resource {
...
 				Optional: true,
 				ForceNew: true,
 			},
-
 			"created": {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
-
+			"updated": {
```
Comment (outcatcher) [inline opentelekomcloud/resource_opentelekomcloud_rds_instance_v3.go]: Same as  previous

**Review by outcatcher — APPROVED** [review]

**Review by outcatcher — CHANGES_REQUESTED** [review]
Approved by mistake, see previous comments

**Review by anton-sidelnikov — APPROVED** [review]

**Review by outcatcher — APPROVED** [review]

**Pull request merged**